miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppDisplay.kt

```diff
@@ -1,12 +1,16 @@
 package com.rakuten.tech.mobile.miniapp
 
 import android.view.View
+import androidx.lifecycle.LifecycleObserver
 
 /**
  * This represents the contract by which the host app can interact with the
  * display unit of the mini app.
+ * This contract complies to Android's [LifecycleObserver] contract, and when made to observe
+ * the lifecycle, it automatically clears up the view state and any services registered with.
```

Comment:
Can you add a short code example to the User Guide of how to use this? Maybe it can go under an "Advanced" or "FAQ" section?

Reply:
Sure, just added one.

Reply:
Thanks @corycaywood..! I think it is good to update the guide.